src/io/sarnowski/swagger1st/parser.clj

```diff
@@ -204,7 +204,9 @@
   (let [items-definition (get definition "items")
         items-parser (create-value-parser items-definition path parser-options)]
     (fn [value]
-      (let [vals (if value (split-array definition value) [])
+      (let [vals (if (and value (not= "body" (first path)))
+                   (split-array definition value)
+                   [])
```

Comment:
Wouldn't this set `vals` to the empty vector when parsing the body? I guess we'd rather want something like this, wouldn't we?

```clojure
(cond
  (= "body" (first path)) value
  (some? value) (split-array definition value)
  :else [])
```